.github/CODEOWNERS

```diff
@@ -37,9 +37,9 @@
 /dns/**/*.py                           @GoogleCloudPlatform/python-samples-owners
 /endpoints/**/*.py                     @GoogleCloudPlatform/python-samples-owners
 /error_reporting/**/*.py               @GoogleCloudPlatform/observability-devx @GoogleCloudPlatform/python-samples-owners
-/eventarc/**/*.py                      @GoogleCloudPlatform/cdpe-serverless @GoogleCloudPlatform/python-samples-owners
+/eventarc/**/*.py                      @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-owners
 /firestore/**/*.py                     @GoogleCloudPlatform/cloud-native-db-dpes @GoogleCloudPlatform/python-samples-owners
-/functions/**/*.py                     @GoogleCloudPlatform/cdpe-serverless @GoogleCloudPlatform/python-samples-owners
+/functions/**/*.py                     @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-owners
 /functions/spanner/*.py                @GoogleCloudPlatform/api-spanner-python @GoogleCloudPlatform/python-samples-owners
```

Comment:
probably out of scope, but if /functions/spanner is owned by the spanner team, can we move it to spanner/functions? Nested ownership is likely to create maintenance and reporting friction. I also note this sample is owned by a different group than the the owner of /spanner.

Reply:
We should update `/spanner/*` to @GoogleCloudPlatform/api-spanner-python as well, but I support moving `/functions/spanner` to `/spanner/functions` if possible, since currently we'd have to manually triage properly for issues in those files, only PRs are taken well care of due to the directory structure.

Reply:
Out of scope for this PR but I'll open an issue for both of these #6763